rakefile

```diff
@@ -267,30 +267,50 @@ rototiller_task :acceptance do |t|
 end
 
 desc 'Run Scale setup and test for gatling'
-rototiller_task :autoscale do |t|
+rototiller_task :autoscale do
   ENV['ENVIRONMENT_TYPE'] = 'gatling'
   ENV['BEAKER_TESTS'] = ENV['BEAKER_TESTS'] || 'tests/Scale.rb'
   ENV['BEAKER_PRESERVE_HOSTS'] = 'always'
   ENV['PUPPET_SCALE_CLASS'] = 'role::by_size::small'
+  ENV['SCALE_TUNE'] = ENV['SCALE_TUNE'] || 'true'
   Rake::Task["performance"].execute
+  Rake::Task["autoscale_copy_log"].execute unless ENV['BEAKER_TESTS'] == ''
 end
 
 desc 'Run Scale setup for gatling'
-rototiller_task :autoscale_setup do |t|
+rototiller_task :autoscale_setup do
   ENV['BEAKER_TESTS'] = ''
   Rake::Task["autoscale"].execute
 end
 
+desc 'Run puppet infrastructure tune setup for gatling'
+rototiller_task :autoscale_tune do
+  ENV['ENVIRONMENT_TYPE'] = 'gatling'
+  ENV['BEAKER_TESTS'] = 'setup/install_gatling/50_tune/10_puppet_infrastructure_tune.rb'
+  ENV['BEAKER_PRESERVE_HOSTS'] = 'always'
```

Comment:
feels like this shouldn't be so absolute... Like if I ran a sim with it set to on_fail... or something you would have just overwrote it.  Maybe some kind of conditional here...  But it depends on the entry points that can get here...

Reply:
I was trying to prevent the user from accidentally de-provisioning the hosts after a long scale run by forgetting to set this correctly. I've updated it to the following:
```
ENV['BEAKER_PRESERVE_HOSTS'] = ENV['BEAKER_PRESERVE_HOSTS'] || 'always'
```